nvflare/fuel/hci/client/event.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
we already have "event.py" in the nvflare/private for the server and client event firing. Better to rename this differently to avoid confusion?

Reply:
I think this is fine. These two are for different sides and never will be used together.